drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/SettingsPresenter.java

```diff
@@ -89,6 +89,15 @@ public void onSaveButton(String scenarioType) {
         saveCommand.execute();
     }
 
+    @Override
+    public void reset() {
```

Comment:
@yesamer 
please create a "reset" method inside the view, and move this code inside it - take note of the above comment about declaring "reset" method in SubDockView interface

Reply:
Done